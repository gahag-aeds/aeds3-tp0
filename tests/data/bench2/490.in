490 245
63 76 35 21 95 8 18 88 27 60 61 38 73 78 72 17 99 36 58 62 0 82 24 2 79 43 12 55 57 83 20 73 59 7 94 6 15 12 95 94 25 8 85 50 38 57 67 37 93 77 51 46 11 75 48 90 70 12 45 28 47 17 1 59 76 47 65 91 60 12 38 37 20 23 87 10 80 7 99 26 36 2 72 48 78 72 38 0 36 36 80 84 5 33 95 82 81 60 25 93 73 63 30 45 86 17 8 19 76 7 45 13 10 69 61 40 41 51 40 77 87 73 13 45 6 8 27 87 21 52 80 46 68 62 43 54 32 51 25 8 59 70 73 21 39 34 61 32 38 53 10 77 26 75 22 85 36 1 72 57 6 5 55 74 67 98 80 99 50 6 60 61 76 33 82 68 20 95 0 58 0 62 35 79 90 10 64 26 63 88 35 69 93 90 95 61 88 76 12 38 82 72 51 10 6 85 30 78 32 83 36 33 45 23 64 35 85 28 61 49 16 96 18 62 86 14 75 27 42 87 17 76 60 69 38 18 6 69 96 39 52 84 24 49 59 88 37 45 68 98 94 36 47 64 98 85 30 73 64 24 61 82 0 73 3 39 43 9 60 91 0 64 75 24 13 34 64 50 79 84 1 25 21 0 42 19 37 72 93 2 49 6 84 49 79 39 88 74 48 0 65 49 64 40 25 78 26 90 80 58 74 81 83 47 33 25 67 71 50 12 73 99 18 9 0 49 48 41 23 48 41 88 49 6 80 75 36 6 17 68 64 43 50 0 91 35 77 58 6 79 22 31 78 40 92 31 89 92 72 12 41 13 52 90 71 32 17 59 90 34 28 7 30 30 7 21 65 84 31 72 64 53 55 94 45 48 77 34 40 49 98 33 15 50 76 38 34 93 98 76 80 78 83 62 8 42 83 25 79 66 49 95 71 5 41 16 5 19 2 97 68 0 83 35 2 59 74 88 4 24 16 36 2 52 98 62 94 33 39 25 51 89 20 22 46 62 90 51 33 92 0 1 44 83 89 98 42 63 38 99 39 55 35 93 7 86 55 53 19 94 31 71 35 51 45 81 65 36 32 98 80 33 52 77 68 41 27 11 56 66 
Add 301 463
Min 49 211
Sub 62 224
Sum 228 390
Add 242 404
Min 214 376
Sub 2 164
Min 181 343
Sub 291 453
Max 212 374
Sub 123 285
Max 137 299
Add 293 455
Max 98 260
Add 280 442
Min 211 373
Add 289 451
Max 194 356
Sub 76 238
Max 287 449
Add 204 366
Sum 207 369
Add 327 489
Sum 9 171
Sub 323 485
Max 50 212
Sub 87 249
Max 326 488
Sub 277 439
Max 248 410
Add 30 192
Max 158 320
Add 296 458
Max 326 488
Sub 59 221
Min 118 280
Sub 212 374
Sum 60 222
Sub 298 460
Max 176 338
Sub 179 341
Sum 93 255
Sub 312 474
Sum 144 306
Sub 98 260
Min 172 334
Sub 263 425
Min 295 457
Sub 274 436
Sum 246 408
Sub 254 416
Sum 150 312
Add 33 195
Max 161 323
Sub 66 228
Min 28 190
Sub 118 280
Max 152 314
Add 27 189
Sum 303 465
Sub 307 469
Sum 291 453
Add 133 295
Max 275 437
Add 290 452
Max 191 353
Sub 300 462
Sum 174 336
Add 251 413
Sum 270 432
Add 257 419
Sum 102 264
Add 270 432
Sum 150 312
Sub 154 316
Max 41 203
Sub 229 391
Sum 324 486
Add 8 170
Min 175 337
Add 242 404
Min 169 331
Sub 232 394
Max 275 437
Add 237 399
Sum 297 459
Add 210 372
Min 262 424
Add 29 191
Max 236 398
Add 237 399
Max 8 170
Sub 108 270
Min 43 205
Add 190 352
Min 70 232
Add 233 395
Sum 162 324
Add 151 313
Min 64 226
Sub 12 174
Sum 81 243
Add 74 236
Sum 282 444
Sub 138 300
Max 227 389
Add 230 392
Sum 39 201
Sub 223 385
Sum 237 399
Sub 214 376
Sum 45 207
Sub 313 475
Max 26 188
Add 227 389
Min 130 292
Sub 323 485
Max 110 272
Add 64 226
Sum 24 186
Sub 253 415
Min 301 463
Sub 32 194
Sum 33 195
Add 16 178
Max 221 383
Sub 102 264
Sum 156 318
Sub 291 453
Max 161 323
Sub 128 290
Max 302 464
Sub 149 311
Sum 201 363
Add 256 418
Min 286 448
Add 8 170
Max 158 320
Add 324 486
Sum 138 300
Add 302 464
Min 118 280
Add 182 344
Sum 195 357
Add 144 306
Min 82 244
Add 324 486
Max 47 209
Add 191 353
Min 61 223
Add 70 232
Max 116 278
Add 269 431
Sum 9 171
Add 57 219
Max 284 446
Sub 138 300
Sum 66 228
Add 20 182
Max 101 263
Sub 226 388
Sum 147 309
Add 75 237
Max 47 209
Add 255 417
Max 239 401
Add 5 167
Min 262 424
Sub 69 231
Sum 237 399
Sub 72 234
Sum 279 441
Add 262 424
Sum 162 324
Sub 146 308
Min 313 475
Sub 243 405
Sum 51 213
Sub 32 194
Max 14 176
Sub 111 273
Min 10 172
Add 37 199
Max 287 449
Sub 253 415
Min 1 163
Sub 151 313
Sum 63 225
Add 67 229
Max 170 332
Add 163 325
Min 292 454
Sub 225 387
Max 146 308
Add 12 174
Max 152 314
Add 57 219
Max 251 413
Add 322 484
Sum 33 195
Sub 68 230
Min 301 463
Sub 311 473
Min 238 400
Sub 135 297
Max 38 200
Add 131 293
Sum 285 447
Sub 88 250
Max 71 233
Sub 206 368
Min 106 268
Add 80 242
Min 151 313
Sub 300 462
Max 5 167
Add 151 313
Sum 123 285
Add 303 465
Min 217 379
Sub 292 454
Min 46 208
Add 182 344
Min 97 259
Sub 191 353
Max 101 263
Add 157 319
Min 247 409
Add 259 421
Min 151 313
Sub 188 350
Min 235 397
Sub 124 286
Min 79 241
Add 145 307
Max 167 329
Sub 116 278
Max 275 437
Sub 124 286
Sum 111 273
Add 18 180
Sum 3 165
Add 124 286
Max 5 167
Sub 153 315
