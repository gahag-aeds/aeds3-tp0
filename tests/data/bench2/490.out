0
8216
1
-1
100
98
98
0
103
103
8604
7453
96
102
103
99
102
-4
7383
102
7659
7813
-6
-1
8125
7779
97
-6
95
8065
7787
97
97
7724
7767
7521
7966
97
8171
-6
-6
100
8272
-1
100
99
-6
-6
8278
-5
7411
8506
103
7798
8732
7633
99
-5
98
7680
-3
7670
101
8151
100
100
7772
-4
96
7483
-8
8143
-7
101
-7
98
7913
105
7763
97
8111
105
105
0
8918
8809
8445
0
7887
103
-5
105
-4
7612
104
0
96
96
104
8438
1
-4
106
8666
106
-8
-7
106
7451
-4
-6
-6
97
-3
-5
-3
-6
102
103
7364
8332
107
